src/Utilities/functions.jl

```diff
@@ -379,16 +390,15 @@ then `isapprox_zero(f)` is `false` but `isapprox_zero(MOIU.canonical(f))` is
 """
 function isapprox_zero end
 
-function isapprox_zero(f::MOI.ScalarAffineFunction, tol)
-    isapprox_zero(f.constant, tol) && all(t -> isapprox_zero(t, tol),
-                                           f.terms)
+isapprox_zero(α::AbstractFloat, tol) = -tol <= α <= tol
```

Comment:
Maybe a separate PR, but this should be named `is_approx_zero`?

Reply:
This is best discussed in a separate issue but I would argue that isapprox_zero has the advantage to be consistent with Base.isapprox